src/devices/Qmc5883l/FieldRange.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Iot.Device.Qmc5883l
+{
+    /// <summary>
+    /// Field ranges of the magnetic sensor can be selected through the register RNG. The full scale field range is
+    /// determined by the application environments.For magnetic clear environment, low field range such as +/- 2gauss
+    /// can be used.The field range goes hand in hand with the sensitivity of the magnetic sensor. The lowest field range
+    /// has the highest sensitivity, therefore, higher resolution.
+    /// </summary>
+    [Flags]
+    public enum FieldRange : byte
+    {
+       /// <summary>
+       /// Field range of 2 Gauss
+       /// </summary>
+        GAUSS_2 = 0x00,
```

Comment:
For public enums, we use a naming convention that is Camel oriented.
So, this enum will be: `Gauss2`, the next one `Gauss8`
We do accept other conventions for internal, register based enums to be close to the documentation and help in debugging. 
This comment applies to all the enums in the same condition.